### ISSUE TITLE
Recovering wallet

### ISSUE THREAD

**Issue by mattkinghtx** [darkwallet/darkwallet#261]
Good afternoon, I realised I have some old funds in a Darkwallet and I'm having no luck recovering it. I have it loaded up and restored the seed, however it's not connecting and I can't find where it is exporting my keys. I was hoping I could retrieve the private keys at least but I'm having no luck.

**Comment by grnqrtr** [comment]
Your post is almost half a year old, were you able to figure out your problem? I don't know if I could help or not, but I help people with bitcoin recover/problems in my free time. All my contact info can be found at www.grnqrtr.com

**Comment by mattkinghtx** [comment]
Yes sir thank you for your help I ended up accessing it though some of your
old posts and tutorials. What's your paypal?

**Comment by grnqrtr** [comment]
Don't worry about it, glad you got it figured out! :)

**Comment by mattkinghtx** [comment]
That works, i appreciate your time!

**Comment by UncipheredLLC** [comment]
@grnqrtr does good work. :)
